### PR TITLE
Add `gulp_create_config` to make gulp.config.js file creation optional, fix #114

### LINKED ISSUE #114: Add option in Gulp role to create config files
Adding the Gulp role always creates a gulp.config.json file, which might not be wanted if the project already uses Gulp with a different structure. We should add an option like `gulp_create_config` that defines whether the config file should be created if inexistent or not.

### PR DESCRIPTION
**Pull request by LeBenLeBen**
* This PR is an improvement related to #114 

- [x] Documentation is written
- [x] [Changelog](https://github.com/liip/drifter/blob/master/CHANGELOG.md) was updated

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -10,6 +10,7 @@ For migration information, you can always have a look at https://liip-drifter.re
 
 ### Added
 - Virtualenv role: add `pip_requirements_dir` option to automatically compile `.in` requirements
+- Gulp: add `gulp_use_config` to make gulp.config.js file creation optional
```
Comment (sephii) [inline CHANGELOG.md]: What do you think about renaming this parameter to `gulp_create_config`?

Comment (LeBenLeBen) [inline CHANGELOG.md]: Done

File: docs/roles/frontend.rst [hunk 2]
```diff
@@ -33,6 +33,8 @@ Parameters
    created, defaults to ``<gulp_directory>/package.json``
 -  **gulp\_package\_json\_author**: Author that should be put in the
    package.json file, defaults to ``Liip AG``
+-  **gulp\_use\_config**: Create the gulp.config.js used by the default Gulpefile.js, defaults to
```
Comment (sephii) [inline docs/roles/frontend.rst]: 🙈

File: provisioning/roles/gulp/tasks/main.yml [hunk 3]
```diff
@@ -3,7 +3,12 @@
   with_items:
     - { src: package.json, dest: "{{ gulp_package_json_path }}" }
     - { src: Gulpfile.js, dest: "{{ gulp_directory }}/Gulpfile.js" }
+
+- name: Create Gulp config file
+  template: src={{ item.src }} dest={{ item.dest }} force=no
+  with_items:
```
Comment (sephii) [inline provisioning/roles/gulp/tasks/main.yml]: Could you please remove the `with_items` and directly put the parameters in the `template` element?

**Comment by LeBenLeBen** [comment]
All fixed!

**Comment by sephii** [comment]
Thanks! 🍋

**Pull request merged**